### DISCUSSION TITLE
What is valid for "Specifics input" to add a book (or anything else?

### DISCUSSION THREAD

**Discussion by TheBig-O** [IgnisDa/ryot#637] [Q&A] [answered]
I'm struggling to figure out what fields are required to be able to add a book. Everything I've tried results in an "Application Error." Any help would be fantastic. 
Is it specific fields, urls, what?
Thank You!

**Comment by IgnisDa** [comment, marked as answer]
You can check the graphql endpoint. For books:
- Go to endpoint linked above the input (for eg: https://ryot.fly.dev/backend/graphql)
- Search for `BookSpecifics`
- 
![image](https://github.com/IgnisDa/ryot/assets/60938164/0104ec52-2ce7-4d6f-ae61-001e4bf71dc4)
- So the JSON would look like this: 
![image](https://github.com/IgnisDa/ryot/assets/60938164/10794283-f9fc-4c56-9029-7effc62f8692)
